Decompiler/Decompiler.cs

```diff
@@ -378,7 +378,7 @@ private void ProcessFile(string path, Stream stream, string originalPath = null)
                     if (OutputFile == null)
                     {
                         // Test extraction code flow while collecting stats
-                        using FileExtract.Extract(resource);
+                        FileExtract.Extract(resource).Dispose();
```

Comment:
`using var _ =  FileExtract.Extract(resource);` is how you do it.

Your dispose won't be called if extract throws.